### PR TITLE
[#13974] add sqlite to .travis.yml

### PR DESCRIPTION
**Pull request by marutosi**
https://www.openproject.org/work_packages/13974

### CODE REVIEW

**Comment by TeatroIO** [comment]
I've prepared a stage to preview changes. <a href='http://marutosi-finn-sqlite3.opf-openproject-2af22bf12c7dfdad4ddf.ttrcloud.com' target='_blank'>Open stage</a> or <a href='http://marutosi-finn-sqlite3.opf-openproject-2af22bf12c7dfdad4ddf.ttrcloud.com/teatro.log' target='_blank'>view logs</a>.

**Comment by marutosi** [comment]
https://community.openproject.org/work_packages/13974

File: .travis.yml [hunk 1]
```diff
@@ -49,6 +49,10 @@ env:
     - "TEST_SUITE=karma"
     - "TEST_SUITE=protractor"
     # mysql2
```
Comment (myabc) [inline .travis.yml]: @marutosi this should read `sqlite3`

**Comment by ulferts** [comment]
Closing as broken and I don't see any effort of getting it to work.

**Pull request closed without merging**